server/src/main/scala/org/http4s/server/middleware/Metrics.scala

```diff
@@ -39,7 +39,7 @@ object Metrics {
       classifierF: Request[F] => Option[String] = { _: Request[F] =>
         None
       }
-  )(routes: HttpRoutes[F])(implicit F: Effect[F], clock: Clock[F]): HttpRoutes[F] =
+  )(routes: HttpRoutes[F])(implicit F: Sync[F], clock: Clock[F]): HttpRoutes[F] =
```

Comment:
This will break MiMa.  I think it got picked up when you changed branches.

Reply:
What's the best way to resolve this? Change the target branch of this PR to master, or create a new PR with the correct source branch?

Reply:
Target master won't get it into the 0.20 series.  Which we're about to release, but since you're still awake, we'll squeeze this in.

You can `git rebase -i` against series/0.20 and drop the two commits that aren't yours and force push, or you can open a new PR.  Whatever's easier for you.

Reply:
Thank you, I have rebased and force pushed.